app/app.py

```diff
@@ -33,6 +35,9 @@ def __init__(self):
             self.db = DB(config)
             self.huey = RedisHuey(host="redis", result_store=False)
             self.web3 = Web3(HTTPProvider(config.HTTP_PROVIDER_URL))
+            
+            fd_config=json.loads(urllib.request.urlopen(config.FRONTEND_CONFIG_FILE).read().decode('utf-8'))
```

Comment:
We should use [requests](https://docs.python-requests.org/en/master/) to simplify fetching network things. For example, it can [spare us](http://docs.python-requests.org/en/master/user/quickstart/#json-response-content) needing to parse JSON explicitly.

Reply:
For this PR I would add the following:

* Switch to requests
* Reload tokens from config file every X minutes

I would not try to solve forkdelta/tokenbase#124 with this to limit the number of changes.

Reply:
For this PR I would add the following:

* Switch to requests
* Reload tokens from config file every X minutes

I would not try to solve forkdelta/tokenbase#124 with this to limit the number of changes.